docs/up-ga/reads/v1.md

```diff
@@ -11,13 +11,69 @@
 <div id="value-display"></div>
 <strong>Description:</strong>
 <div class="tile-1" style="text-align:center; color:black">
-No data available for the specified measure.
+  Total reads on pages where the work can be read, as determined by Google Analytics. To count multiple reads on the same page by the same user, there must be a gap of at least 30 minutes from the last read of that page, even if that previous read was not counted.
```

Comment:
I really think this should be "Total views on pages where the work can be read".

Am happy to discuss tomorrow